package.json

```diff
@@ -1,45 +1,62 @@
 {
-  "name": "ovh-angular-mondial-relay",
+  "name": "@ovh-ux/ng-ovh-mondial-relay",
   "version": "4.1.1",
   "description": "Display a map to select mondial relay",
+  "keywords": [
+    "map",
+    "mondial",
+    "relay"
+  ],
+  "repository": "ovh-ux/ng-ovh-mondial-relay",
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/ng-ovh-mondial-relay.js",
   "scripts": {
-    "test": "grunt test"
+    "build": "rollup -c",
+    "dev": "rollup -c --watch",
+    "lint": "run-p lint:html lint:css lint:js lint:md",
+    "lint:css": "stylelint 'src/**/*.less' --fix",
+    "lint:html": "htmlhint 'src/**/*.html'",
+    "lint:js": "eslint --quiet --fix --format=pretty src",
+    "lint:md": "eslint --quiet --fix --format=pretty --ext .md .",
+    "prepare": "rollup -c",
+    "test": "yarn run lint",
+    "version": "npx conventional-changelog-cli -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
+    "postversion": "git push && git push --tags"
+  },
+  "dependencies": {
+    "bootstrap": "~3.3.7",
+    "leaflet": "0.7.x",
+    "lodash": "^4.17.11",
+    "ovh-ui-kit": "^2.23.1",
+    "ovh-ui-kit-bs": "^2.1.1"
   },
-  "main": "./dist/ovh-angular-mondial-relay.min.js",
-  "repository": "ovh-ux/ovh-angular-mondial-relay",
   "devDependencies": {
-    "eslint": "^4.0.0",
-    "eslint-config-ovh": "^0.1.1",
-    "grunt": "0.4.x",
-    "grunt-angular-templates": "^0.5.4",
-    "grunt-autoprefixer": "~2.2.0",
-    "grunt-base64": "0.1.0",
-    "grunt-bump": "0.3.1",
-    "grunt-complexity": "~0.1.0",
-    "grunt-contrib-clean": "~0.6.0",
-    "grunt-contrib-concat": "~0.5.0",
-    "grunt-contrib-copy": "~0.7.0",
-    "grunt-contrib-csslint": "~0.4.0",
-    "grunt-contrib-cssmin": "~0.12.2",
-    "grunt-contrib-less": "~1.0.0",
-    "grunt-contrib-uglify": "~0.8.0",
-    "grunt-contrib-watch": "~0.6.1",
-    "grunt-css-url-embed": "1.6.1",
-    "grunt-eslint": "^20.0.0",
-    "grunt-karma": "~0.12.0",
-    "grunt-ng-annotate": "~0.10.0",
-    "grunt-ng-constant": "1.1.0",
-    "grunt-ngdocs": "^0.2.5",
-    "grunt-ovh-translation": "^2.1.1",
-    "jasmine-core": "^2.4.1",
-    "karma": "~0.13.0",
-    "karma-jasmine": "~0.3.0",
-    "karma-ng-html2js-preprocessor": "~0.1.0",
-    "karma-phantomjs-launcher": "~1.0.0",
-    "karma-spec-reporter": "0.0.26",
-    "matchdep": "^0.3.0",
-    "phantomjs-prebuilt": "^2.1.7"
+    "@commitlint/cli": "^7.1.2",
+    "@commitlint/config-angular": "^7.1.2",
+    "@ovh-ux/component-rollup-config": "ovh-ux/component-rollup-config#feat/image",
```

Comment:
https://github.com/ovh-ux/component-rollup-config/blob/master/CHANGELOG.md#500-beta9-2019-02-07

Reply:
`ovh-ux/component-rollup-config` bumped to `5.0.0-beta.9`